src/modules/integrator-guide/pages/compose-configuration/index.adoc

```diff
@@ -864,3 +864,43 @@ You can use this to create context-aware validators.
 ----
 length(title) < 5
 ----
+
+== Opening Records in Modal Windows
+
+Record lists have an option to open records in a modal on record click.
+By default, records are opened in a new page.
+When set, this option allows users to inspect specific records, while staying on the original page.
+
+=== Configuring the Option
+
+To enable the option of opening records in a modal, navigate to the page builder, choose the record list block and go to its record list tab. 
+Scroll down to the Records section and you will find the option in the "On record click" select dropdown.
+
+[annotation,role="data-zoomable"]
+----
+{
+  "image": {
+    "rel": "compose-configuration/enabling-record-in-modal.png",
```

Comment:
If you wanna go a step further you could define a focus around this area

![image](https://user-images.githubusercontent.com/12545711/233565989-2365bffc-efda-4b1f-bb2f-7ac4634af8c2.png)

This would then blur everything except for the records section (which is the only bit we care about).
I sometimes use this when I want to reduce distractions.